Chess in Python.py

```diff
@@ -138,7 +171,10 @@ class GameState: # GameState stores all relevant attributes of the given game's
     print("This game is called chess. There are two players; White (aka W) and Black (aka B). Each player starts with the same number and type of pieces.\n")
     print(f"It is {str(game.whichTurn)}'s turn.\n Here is the game board: \n")
     print(game.gameBoard)
-    userPieceChoice = askForValidPiece(f"It's {str(game.whichTurn)}'s turn") # ask the player for the x and y values of the piece which they'd like to move.
-    print(f"you selected this piece: {userPieceChoice}")
 
-    break    # exit out of the Driver code
\ No newline at end of file
+    # ask the player for the x and y values of the piece which they'd like to move from and to.
+    move = askForMove(f"It's {str(game.whichTurn)}'s turn")
+    userSelection = game.gameBoard.getPiece(*(move[0].getXY()))
```

Comment:
@mknowles36 on line 176, is the asterisk being used here to unpack the tuple of ints which the .getxy() method returns? If not, what is the asterisk doing?

Reply:
Yup - that's exactly what the asterisk is for.